calico/operations/ebpf/enabling-ebpf.mdx

```diff
@@ -51,15 +51,11 @@ eBPF (or "extended Berkeley Packet Filter"), is a technology that allows safe mi
 
 - IPv6
 
-:::note
-This feature is tech preview. Tech preview features may be subject to significant changes before they become GA.
-:::
-
 Limitations:
 
-  - Dual stack is not supported. However, ipv4 traffic is allowed into hosts (not workloads) because many managed Kubernetes environments have ipv4-based control plane.
-  - `doNotTrack` policies (xdp) are not supported
-  - IPIP is not supported ({{prodname}} iptables does not support it either). VXLAN is the recommended overlay for eBPF mode.
+  - IPv6 only hosts are not supported.
```

Comment:
I'm a bit confused about what's supported. Above we say IPv6 is supported, and here we say it isn't. 

Is it better to specify that we support IPv4-only and dual-stack?

Reply:
> Is it better to specify that we support IPv4-only and dual-stack?

that sounds better, I also got confused

Reply:
we are working on this limitation to go away

Reply:
Ok, let's change as follows, starting with line 52:

```
- IPv4-only and dual-stack hosts

Limitations:

- IPv6-only hosts are not supported.